src/main/scala/com/microsoft/hyperspace/index/serde/LogicalPlanSerDeUtils.scala

```diff
@@ -170,6 +179,11 @@ object LogicalPlanSerDeUtils {
           output,
           catalogTable,
           isStreaming) =>
+        val format = fileFormat match {
+          case f: ParquetFileFormat => f
+          case CSVFileFormatWrapper => new CSVFileFormat
+          case _ => throw HyperspaceException("Unsupported File Format found.")
```

Comment:
Can we check few more types? https://microsoft.github.io/hyperspace/docs/toh-introduction/

Reply:
added checks and support for json and orc formats